src/earn/poolInfoScreen/EarnPoolInfoScreen.tsx

```diff
@@ -546,7 +546,7 @@ const styles = StyleSheet.create({
     flexDirection: 'row',
     alignItems: 'center',
     justifyContent: 'center',
-    marginBottom: Spacing.Thick24,
+    padding: variables.contentPadding,
```

Comment:
not a fan of `variables.contentPadding`, as we have different padding for different elements throughout the app. Can we not use this? or define a consistent way of using this (e.g., have a clear definition of when to use this vs directly using Spacing)? At this point this just seems a proxy for a specific Spacing value.

Reply:
Switch to Spacing in 26d57378f43b80076c56bbc0aaa75059759ef423.